scylladb/sphinx-scylladb-theme
# Make poetry a prerequisite

Closes #307 & closes #239

## How to test this PR

1. Clone this PR.

2. Delete poetry (old version) from your computer:

```
curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | POETRY_UNINSTALL=1 python -
```

3. Install the latest version of poetry:

```
curl -sSL https://install.python-poetry.org | python -
```

> For more detailed instructions, see https://python-poetry.org/docs/master/

4. Move to the docs folder, and run ``make preview``.

5. The documentation should build without errors.
